python/lsst/ts/observatory/control/maintel/mtcalsys.py

```diff
@@ -905,41 +909,56 @@ async def calculate_optimized_exposure_times(
         -------
         exposure_list : `list`[ATCalsysExposure|MTCalsysExposure]
             List of exposure information, includes wavelength
-            and camera, fiberspectrograph and electrometer exposure times.
+            and camera, fiberspectrograph and electrometer exposure times,
+            and LED dac value.
         """
         exposures: list[MTCalsysExposure] = []
         for wavelength in wavelengths:
 
-            electrometer_exptimes = await self._calculate_electrometer_exposure_times(
-                exptimes=config_data["exposure_times"],
-                electrometer_integration_time=config_data[
-                    "electrometer_integration_time"
-                ],
-                use_electrometer=config_data["use_flatfield_electrometer"]
-                or config_data["use_cbp_electrometer"],
-            )
-            fiberspectrograph_exptimes_red = (
-                await self._calculate_fiberspectrograph_exposure_times(
-                    exptimes=config_data["exposure_times"],
-                    use_fiberspectrograph=config_data["use_fiberspectrograph_red"],
+            for exptime in config_data["exposure_times"]:
+                dac = config_data["dac_value"]
+                if config_data["ptc"]:
+                    if exptime < 1.0:
+                        dac = 0.15
+                        exptime = exptime * 100.0
+                    elif exptime > 30.0:
```

Comment:
This could also result in an exptime that is negative or less than 1s. As written, if exptime=31s, then it would change to -9 sec. Also if it is 40.1s, it would become 0.1s. I don't think either of those are what we want.